bugalink-frontend/hooks/useReviews.ts

```diff
@@ -5,7 +5,7 @@ export default function useReviews(driverId) {
   // TODO: once we have local sessions, we can extract the user id from the session
 
   const { data, error, isLoading } = useSWR(
-    `/api/users/${driverId}/reviews`,
+    `/users/${driverId}/reviews/rating`,
```

Comment:
Same, I don't know if this endpoint still exists. Backend guys made the rating system yesterday, check swagger for endpoints